### DISCUSSION TITLE
[ 问题咨询 ]open api页面打不开，静态资源加载失败

### DISCUSSION THREAD

**Discussion by weizhenyou007** [hhyo/Archery#1613] [Q&A]
### 问题描述
docker-compose部署的服务open api页面打不开，静态资源加载失败
https://cdn.jsdelivr.net/npm/swagger-ui-dist@latest/swagger-ui.css
https://cdn.jsdelivr.net/npm/swagger-ui-dist@latest/swagger-ui-bundle.js
https://cdn.jsdelivr.net/npm/swagger-ui-dist@latest/swagger-ui-standalone-preset.js
https://cdn.jsdelivr.net/npm/swagger-ui-dist@latest/favicon-32x32.png

### 版本信息
 - 应用版本/分支：Release v1.8.5  
 - 部署方式：Docker-compose

需要通过api调用审核平台接口，我应该配置哪些内容？如何解决静态资源加载不到的问题

**Comment by nick2wang** [comment]
api使用参考：https://github.com/hhyo/Archery/wiki/api
swagger使用本地静态文件：#1498

**Reply by zhongjiazhen2024** [reply]
<img width="685" alt="2534188c3b444a9dea70fbd56b7a673c" src="https://github.com/hhyo/Archery/assets/170154139/31fdac79-aebd-4b8c-92f5-11addd2711c0">

调用API失败，请问是什么问题啊？
